src/useGet.ts

```diff
@@ -94,26 +95,22 @@ export default <CustomApplication extends Application>(feathers: CustomApplicati
       isLoading.value = false;
     };
 
-    watch(_id, async () => {
-      await get();
-    });
-
-    const connectListener = () => {
+    const load = () => {
       void get();
     };
 
-    feathers.on('connect', connectListener);
+    const unload = () => {
+      unloadEventHandlers();
+      feathers.off('connect', load);
+    };
 
-    onMounted(async () => {
-      await get();
-    });
+    watch(_id, load, { immediate: true });
```

Comment:
`immediate` replaces `onMounted`?

Reply:
THB I am not sure why we loaded on `onMounted` instead of directly anyways.